company/values.md

```diff
@@ -1,15 +1,32 @@
 # Sourcegraph values
 
-Our values are:
+These values are some of the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
 
-## People
+This list isn't intended to cover everything we care about; instead, it lists the values that we frequently find useful and refer to. We'll keep this list up to date with the frequently used beliefs and principles (adding, editing, and removing entries as needed). Our hope is that this makes this list more accurate and useful than if it were a list of stale, vague, aspirational, or obvious values.
 
-Together we are advancing technology for the good of people all around the world. We will attract, hire and retain the best teammates in the world and treat everyone in a first-class manner.
+## High quality
 
-## Journey
+Every person on our team is individually responsible for knowing what high-quality work looks like and producing high-quality work.
```

Comment:
I like this point, but its description seems to be more aimed towards self-management than delivering high quality.